inst/rmarkdown/templates/ams/skeleton/skeleton.Rmd

```diff
@@ -1,211 +1,462 @@
-
-## Equations
-
-Brief equations or terms set inline in text must be set as a single-line expression because page proofs are not double spaced, for example, $\rho^{-1}p/x$ or $(1/{\rho})p/x$ or $(a-b)/(c+d)$; that is, use a superscript $-1$ for the denominator. In case of a more complicated term or equation, it should be set as an unnumbered display equation, such as
-
-$$
-x=\frac{2b\pm\sqrt{b^{2}-4ac}}{2c}.
-$$
-
-Otherwise, numbered display equations can be entered using the appropriate equation command, such as 
-
-\begin{equation}
-x=\frac{2b\pm\sqrt{b^{2}-4ac}}{2c}.  
-\end{equation}
-
-Lists of equations are punctuated as written English, and commas, semicolons, and periods are placed where appropriate. Conjunctions such as "and", "while", "when", or "for" are also typically placed before the final element in a mathematical phrase, as befits the intended mathematical meaning.  
-
-## Figures and tables
-
-The AMS prefers that all figures and tables are placed **at the end of the document** prior to submission. A list of tables and a list of figures will appear near the end of the PDFfile, before the actual tables and figures. These lists are necessary for submission.
-
-For appendix figures and tables, special commands are needed to manually change the numbering to ensure that each appendix figure or table is numbered as part of the respective appendix and not as a continuation of the main paper. Use the command `\appendcaption{}` instead of the usual `caption{}` to adjust the  numbering; for example, for Table A1, you would use the command `\appendcaption{A1}`.
-
-Note that the normal `\ref{}` command cannot be used to cite appendix figures and tables as the numbering will be incorrect. Callouts for appendix figures and tables in the text will need to be written out as plain text, for example, Fig. A1 and Table A1.
-
-### Figures
-
-The insertion of a sample figure (Fig. \ref{f1})  
-and caption is given below (in the .tex document) and at the end of the document. Standard figure sizes are 19 (one column), 27, 33, and 39 (two columns) picas.
-
-\begin{figure}[h]
- \centerline{\includegraphics[width=19pc]{figure01.pdf}}
-  \caption{Enter the caption for your figure here.  Repeat as
-  necessary for each of your figures.}\label{f1}
-\end{figure}
-
-```{r,echo=FALSE,fig.cap='test the rmd output',fig.align='center',fig.width=3.17}
-plot(1:10)
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(fig.path = "",     # AMS required
+                      out.extra = "",    # To force figure labels
+                      echo = FALSE)      # Don't show R code   
 ```
 
-### Tables
-Each table must be numbered, provided with a caption, and mentioned specifically in the text. 
-See below (in the .tex document) and at the end of the document for the formatting of a sample table (Table
-\ref{t1}).
+\section{Introduction}
```

Comment:
We would need to change to Markdown before merging
```suggestion
# Introduction
```

If we are lazy to do it manually, we can probably convert the tex file body using Pandoc directly